package.json

```diff
@@ -136,6 +136,11 @@
           "type": "boolean",
           "default": true,
           "description": "When you run 'Send To Deck' the title (h1) of the markdown file is stored as a tag. This is useful if you have 'daily' notes, you can use the same deck but separate cards by title"
+        },
+        "anki.md.card.notecardIdPattern": {
+          "type": "string",
+          "default": "<!--\\s*?notecardId\\s*?\\[:=\\]\\s*?(\\d+)\\s*?-->",
+          "description": "Text to match match the commented notecard ID."
```

Comment:
Do we need this to be configurable? I have a feeling if people change this it could become messy. 
I'd start off with it being hardcoded

Reply:
Ok, yes I guess it could result in lots of questions/mis-configured regexes (and thus lots of github issues). 

I'm going making this value just exist in code, I'll add it to a constants file if there is one, or just leave it by the parsing logic if there isn't.